dpc/src/testnet1/parameters.rs

```diff
@@ -158,6 +164,7 @@ impl Parameters for Testnet1Parameters {
     dpc_setup!{local_data_crh, LOCAL_DATA_CRH, LocalDataCRH, "AleoLocalDataCRH0"}
     dpc_setup!{program_commitment_scheme, PROGRAM_COMMITMENT_SCHEME, ProgramCommitmentScheme, "AleoProgramIDCommitment0"} // TODO (howardwu): Rename to "AleoProgramCommitmentScheme0".
     dpc_setup!{program_id_crh, PROGRAM_ID_CRH, ProgramIDCRH, "AleoProgramIDCRH0"}
+    dpc_setup!{record_encoding_scheme, RECORD_ENCODING_SCHEME, RecordEncodingScheme, "AleoRecordEncoding0"} // TODO (howardwu): Rename to "AleoRecordEncodingScheme0".
```

Comment:
As a side note, we have a few comments `// TODO (howardwu): Rename to "AleoProgramCommitmentScheme0".`

any plan on when such renaming would be done?

Reply:
Perhaps I can address all of the TODOs for input messages in this PR, given we will need to do a resampling of parameters.